code/src/reducers/fetch.js

```diff
@@ -0,0 +1,45 @@
+// -- reducer
+import { game } from "./game";
+import { ui } from "./ui";
+
+export const fetchGame = () => {
+  return (dispatch, getStore) => {
+    dispatch(ui.actions.setLoading(true));
+
+    fetch("https://wk16-backend.herokuapp.com/start", {
+      method: "POST",
+      body: JSON.stringify({ username: getStore().game.username }),
+      headers: { "Content-Type": "application/json" },
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        dispatch(game.actions.setGame(data));
+
+        dispatch(ui.actions.setLoading(false));
+      });
+  };
+};
+
+//function that returns another function aka THUNKy
+//why not curlies around the direction?
```

Comment:
It's just a regular parameter right? You are not decomposing any object here and therefore no need for curlys we believe.